### PR TITLE
Add back functionality

### PR DESCRIPTION
**Pull request by nicosuave**
Fixes https://github.com/sindresorhus/anatine/issues/5

### CODE REVIEW

**Comment by sindresorhus** [comment]
Sorry, I just pushed a big change to the shortcuts. The previous way was not working correctly. It's not using the Mousetrap library to implement shortcuts: https://github.com/sindresorhus/anatine/blob/4b8e4430ec54f02cfbf73b023521c2b51f3caca1/browser.js#L54 I don't think this needs a menu item anyways.

**Comment by nicosuave** [comment]
No worries. Pushed a revision 👍

File: browser.js [hunk 1]
```diff
@@ -77,6 +77,10 @@ function registerShortcuts(username) {
 			$('button[data-testid="dmComposerSendButton"]').click();
 		}
 	});
+
+	Mousetrap.bindGlobal('backspace', () => {
```
Comment (sindresorhus) [inline browser.js]: Shouldn't be global? This will make backspace in textarea's go back instead of delete a character.

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by nicosuave** [comment]
You got it!

**Pull request closed without merging**